templates/citizen_project/_form.html.twig

```diff
@@ -147,17 +147,38 @@
     </div>
 </div>
 
+<div class="form__new" id="citizen-project-phone">
+    <div class="register__form__phone">
+        {{ form_label(form.required_means, 'Votre téléphone :', {label_attr: {class: 'label__header--large'}}) }}
+        <div class="l__row l__row--top l__row--h-stretch">
+            {{ form_widget(form.phone.country) }}
+            {{ form_widget(form.phone.number, {attr: {placeholder: 'Téléphone' }}) }}
```

Comment:
We should be consistent and always put spaces inside curly braces to improve the readability of hashes (here there is one at the end, but not at the start and not for both).